### ISSUE TITLE
Device support?

### ISSUE THREAD

**Issue by d00001** [poundifdef/VirginMobileMinutesChecker#38]
Why does the description say "For those using the Motorola Triumph, LG Optimus V, or Samsung Intercept"? Is there any reason why this wouldn't work on other devices?

**Comment by pattonj** [comment]
No, it should work for other devices. I tested it on my Nexus 7 and it works ok. The layout/button sizes would need modified, but it still works fine. My guess is that those were the Android phones Virgin Mobile was using/ the author tested it on when he first made the app. Virgin Mobile has added a lot more android phones recently.
